fpop/preprun_fp.py

```diff
@@ -0,0 +1,187 @@
+from dflow import (
+    InputParameter,
+    OutputParameter,
+    Inputs,
+    InputArtifact,
+    Outputs,
+    OutputArtifact,
+    Workflow,
+    Step,
+    Steps,
+    upload_artifact,
+    download_artifact,
+    argo_range,
+    argo_len,
+    argo_sequence,
+)
+from dflow.python import(
+    PythonOPTemplate,
+    OP,
+    OPIO,
+    OPIOSign,
+    Artifact,
+    Slices,
+)
+
+from dflow.plugins.dispatcher import DispatcherExecutor
+from dflow.plugins.lebesgue import LebesgueExecutor
+import os,sys
+from typing import Optional, Set, List
+from pathlib import Path
+from copy import deepcopy
+
+class PrepRunFp(Steps):
+    def __init__(
+        self,
+        name : str,
+        prep_op : OP,
+        run_op : OP,
+        prep_image : str,
+        run_image : str,
+        prep_config : Optional[dict] = None,
+        run_config : Optional[dict] = None,
+        upload_python_packages : Optional[List[os.PathLike]] = None,
+    ):
+        self._input_parameters = {
+            "inputs" : InputParameter(),
+            "type_map" : InputParameter(),
+            "backward_list" : InputParameter(),
+            "config" : InputParameter(type=dict , value={}),
+            "optional_input" : InputParameter(type=dict , value={}),
```

Comment:
Please provide doc explaining what are the acceptable keys in the optional input.